eth/api.go

```diff
@@ -389,6 +389,8 @@ func (api *PublicDebugAPI) AccountRange(blockNrOrHash rpc.BlockNumberOrHash, sta
 		if err != nil {
 			return state.IteratorDump{}, err
 		}
+	} else {
+		return state.IteratorDump{}, fmt.Errorf("must specify either block number or block hash")
```

Comment:
You are not using any parameters in `fmt.Errorf()`, so please use `errors.New()` instead.

Reply:
Furthermore, perhaps `either block number of block hash must be specified` is a better error description IMO

Reply:
Just pushed an update to resolve both